cmd/peer_linux.go

```diff
@@ -16,11 +16,12 @@ package cmd
 import (
 	"context"
 
+	"golang.org/x/sync/errgroup"
+
 	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/vmsock"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"golang.org/x/sync/errgroup"
```

Comment:
The grouping here is odd.
```suggestion
	"github.com/sirupsen/logrus"
	"github.com/spf13/cobra"
	"github.com/spf13/viper"
	"golang.org/x/sync/errgroup"

	"github.com/rancher-sandbox/rancher-desktop-host-resolver/pkg/vmsock"
```